### PR TITLE
update changelog with version v2.19

### PR DESCRIPTION
**Pull request by vgramer**
**What this PR does / why we need it**:
First version of the changelog. Still need to 

- [x] redact part about cve in introduction
- [x] redact part about highlight
- [x] maybe need some makeup too

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,267 @@
...
+- Add PATCH endpoint for external cluster ([#8135](https://github.com/kubermatic/kubermatic/issues/8135))
+- new endpoint to list External MD Metrics: `/api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/machinedeployments/{machinedeployment_id}/nodes/metrics` ([#8239](https://github.com/kubermatic/kubermatic/issues/8239))
+- Manage AKS Clusters natively. Extend external cluster functionality to import existing AKS cluster ([#8244](https://github.com/kubermatic/kubermatic/issues/8244))
+- Create AKS nodePool ([#8399](https://github.com/kubermatic/kubermatic/issues/8399))
+- Upgrade AKS NodePool Kubernetes version  
+  Scale AKS NodePool
+  Get AKS NodePool machineDeployments and nodes
+  Delete AKS NodePool ([#8349](https://github.com/kubermatic/kubermatic/issues/8349))
+- GET `/providers/eks/clusters` => `/projects/{project_id}/providers/eks/clusters` ([#8423](https://github.com/kubermatic/kubermatic/issues/8423))
+- Adds GET `/api/v2/providers/eks/clusters` ([#8045](https://github.com/kubermatic/kubermatic/issues/8045))
```
Comment (imharshita) [inline CHANGELOG.md]: Please remove this as this endpoint is modified here https://github.com/kubermatic/kubermatic/pull/8616/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR50
to `/api/v2/projects/{project_id}/providers/eks/clusters`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,267 @@
...
+- GET `/api/v2/providers/aks/clusters` to list Azure AKS clusters ([#8299](https://github.com/kubermatic/kubermatic/issues/8299))
```
Comment (imharshita) [inline CHANGELOG.md]: same comment as above, modified to `/api/v2/projects/{project_id}/providers/aks/clusters`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,267 @@
...
+### New and Enhanced Features
+
+#### External Cluster
+- endpoint to get external cluster upgrades ([#8115](https://github.com/kubermatic/kubermatic/issues/8115))
+- new endpoint to list external cluster machine deployments ([#8200](https://github.com/kubermatic/kubermatic/issues/8200))
+- Add PATCH endpoint for external cluster ([#8135](https://github.com/kubermatic/kubermatic/issues/8135))
+- new endpoint to list External MD Metrics: `/api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/machinedeployments/{machinedeployment_id}/nodes/metrics` ([#8239](https://github.com/kubermatic/kubermatic/issues/8239))
+- Manage AKS Clusters natively. Extend external cluster functionality to import existing AKS cluster ([#8244](https://github.com/kubermatic/kubermatic/issues/8244))
+- Create AKS nodePool ([#8399](https://github.com/kubermatic/kubermatic/issues/8399))
+- Upgrade AKS NodePool Kubernetes version
```
Comment (imharshita) [inline CHANGELOG.md]: please change this to bullet points, it looks messy this way

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,267 @@
...
+- 1.21.8
+- 1.22.5
+
+
+### New and Enhanced Features
+
+#### External Cluster
+- endpoint to get external cluster upgrades ([#8115](https://github.com/kubermatic/kubermatic/issues/8115))
+- new endpoint to list external cluster machine deployments ([#8200](https://github.com/kubermatic/kubermatic/issues/8200))
+- Add PATCH endpoint for external cluster ([#8135](https://github.com/kubermatic/kubermatic/issues/8135))
```
Comment (imharshita) [inline CHANGELOG.md]: NIT: please add this `PATCH /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}`

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,267 @@
...
+- 1.20.14
+- 1.21.8
+- 1.22.5
+
+
+### New and Enhanced Features
+
+#### External Cluster
+- endpoint to get external cluster upgrades ([#8115](https://github.com/kubermatic/kubermatic/issues/8115))
+- new endpoint to list external cluster machine deployments ([#8200](https://github.com/kubermatic/kubermatic/issues/8200))
```
Comment (imharshita) [inline CHANGELOG.md]: NIT: please add this `GET /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/machinedeployments`

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,267 @@
...
+- 1.20.13
+- 1.20.14
+- 1.21.8
+- 1.22.5
+
+
+### New and Enhanced Features
+
+#### External Cluster
+- endpoint to get external cluster upgrades ([#8115](https://github.com/kubermatic/kubermatic/issues/8115))
```
Comment (imharshita) [inline CHANGELOG.md]: NIT: please add this `GET /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/upgrades`

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,284 @@
+# Kubermatic 2.19
+## [v2.19.0](https://github.com/kubermatic/kubermatic/releases/tag/v2.19.0)
+
+Before upgrading, make sure to read the [general upgrade guidelines](https://docs.kubermatic.com/kubermatic/v2.19/upgrading/guidelines/). Consider tweaking `seedControllerManager.maximumParallelReconciles` to ensure usercluster reconciliations will not cause resource exhausting on seed clusters.
+
+Several vulnerabilities were identified in Kubernetes (CVE-2021-44716, CVE-2021-44717, CVE-2021-3711, CVE-2021-3712, CVE-2021-33910) have been identified and fixed in Kubernetes 1.20.13, 1.21.8, 1.22.5.
```
Comment (SimonTheLeg) [inline CHANGELOG.md]:
```suggestion
Several vulnerabilities were identified in Kubernetes (CVE-2021-44716, CVE-2021-44717, CVE-2021-3711, CVE-2021-3712, CVE-2021-33910) which have been fixed in Kubernetes 1.20.13, 1.21.8, 1.22.5.
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,284 @@
...
+- **Enhanced Control Plane Networking with Konnectivity Support:** To harness the power of Cilium with eBPF, KKP 2.19 comes with added Konnectivity support. It provides TCP level proxy for the control plane (seed cluster) to worker nodes (user cluster) communication. It is based on the upstream [apiserver-network-proxy project](https://github.com/kubernetes-sigs/apiserver-network-proxy) and replaces the older KKP-specific solution based on OpenVPN and network address translation.
+- **Multiple Backup Locations for Improved Business Continuity:** As KKP Admin, you can now configure multiple backup destinations from the UI Admin Panel. Users can select daily, weekly, monthly or customized backup options.
+
+### Breaking Changes
+
+- ACTION REQUIRED: When upgrading from older KKP versions, additional flag --migrate-nginx-ingress is required to perform upgrade of nginx-ingress-controller. Ingress will be briefly unavailable during this process ([#8333](https://github.com/kubermatic/kubermatic/issues/8333))
```
Comment (dermorz) [inline CHANGELOG.md]:
```suggestion
- ACTION REQUIRED: When upgrading from older KKP versions, additional flag `--migrate-nginx-ingress` is required to perform upgrade of nginx-ingress-controller. Ingress will be briefly unavailable during this process ([#8333](https://github.com/kubermatic/kubermatic/issues/8333))
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,284 @@
...
+- 1.22.5
+
+
+### New and Enhanced Features
+
+#### External Cluster
+- endpoint to get external cluster upgrades `GET /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/upgrades` ([#8115](https://github.com/kubermatic/kubermatic/issues/8115))
+- new endpoint to list external cluster machine deployments `GET /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/machinedeployments` ([#8200](https://github.com/kubermatic/kubermatic/issues/8200))
+- Add PATCH endpoint for external cluster `PATCH /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}`([#8135](https://github.com/kubermatic/kubermatic/issues/8135))
+- new endpoint to list External MD Metrics: `/api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/machinedeployments/{machinedeployment_id}/nodes/metrics` ([#8239](https://github.com/kubermatic/kubermatic/issues/8239))
```
Comment (SimonTheLeg) [inline CHANGELOG.md]:
```suggestion
- New endpoint to get external cluster upgrades `GET /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/upgrades` ([#8115](https://github.com/kubermatic/kubermatic/issues/8115))
- New endpoint to list external cluster machine deployments `GET /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/machinedeployments` ([#8200](https://github.com/kubermatic/kubermatic/issues/8200))
- Add PATCH endpoint for external cluster `PATCH /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}`([#8135](https://github.com/kubermatic/kubermatic/issues/8135))
- New endpoint to list External MD Metrics: `/api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/machinedeployments/{machinedeployment_id}/nodes/metrics` ([#8239](https://github.com/kubermatic/kubermatic/issues/8239))
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,284 @@
...
+- **Multiple Backup Locations for Improved Business Continuity:** As KKP Admin, you can now configure multiple backup destinations from the UI Admin Panel. Users can select daily, weekly, monthly or customized backup options.
+
+### Breaking Changes
+
+- ACTION REQUIRED: When upgrading from older KKP versions, additional flag --migrate-nginx-ingress is required to perform upgrade of nginx-ingress-controller. Ingress will be briefly unavailable during this process ([#8333](https://github.com/kubermatic/kubermatic/issues/8333))
+- ACTION REQUIRED: Set the default Service Accounts automountServiceAccountToken property to false for all KKP provided namespaces and kube-system ([#8344](https://github.com/kubermatic/kubermatic/issues/8344))
+- cert-manager chart has been updated to use upstream chart from https://charts.jetstack.io as a dependency, updated to version 1.6.1
```
Comment (dermorz) [inline CHANGELOG.md:27]: There should be a hint that on upgrading KKP with the installer the flag `--migrate-upstream-cert-manager` needs to be set. (analog to the `--migrate-nginx-ingress` hint)

This is mentioned in the docs to upgrade from v2.18 to v2.19, but is missing here in the release notes.

**Review by dermorz — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,3 +1,284 @@
...
+
+
+#### Operating System Management (OSM)
+- Integrating Operating System Manager in KKP ([#8473](https://github.com/kubermatic/kubermatic/issues/8473))
+- Validating webhooks for OSM resources, OperatingSystemProfile and OperatingSystemConfig ([#8503](https://github.com/kubermatic/kubermatic/issues/8503))
+- Deploy OSM CRDs in KKP seed clusters ([#8528](https://github.com/kubermatic/kubermatic/issues/8528))
+- Introduce new field enableOperatingSystemManager in Cluster and ClusterTemplate CRD specification:  
+  - use-osm flag is passed to machine-controller and machine-controller webhooks in case OSM is enabled  
+  - API integration for OSM ([#8523](https://github.com/kubermatic/kubermatic/issues/8523))
+- CRDs for OSP and OSC are updated in kubermatic-operator
```
Comment (SimonTheLeg) [inline CHANGELOG.md]: GH won't let me do suggestions, but this should be one sentence

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,3 +1,284 @@
...
+- Manage GKE Clusters natively. Extend external cluster functionality to import existing GKE cluster ([#8046](https://github.com/kubermatic/kubermatic/issues/8046))
+
+#### KubeVirt
+- Add API endpoints for Kubevirt VirtualMachineInstancePresets and StorageClasses ([#8441](https://github.com/kubermatic/kubermatic/issues/8441))
+- Support for KubeVirt CSI driver ([#8416](https://github.com/kubermatic/kubermatic/issues/8416))
+
+### Nutanix
+- Add experimental/alpha cloud provider support for Nutanix ([#8448](https://github.com/kubermatic/kubermatic/issues/8448))
+
+### Mics
```
Comment (sankalp-r) [inline CHANGELOG.md]: It's a typo. It should be `Misc`

File: CHANGELOG.md (hunk 8 above)
Comment (themue) [inline CHANGELOG.md]: Which "older" KKP versions? Also `--migrate-nginx-ingress` (like other flags).

File: CHANGELOG.md [hunk 13]
```diff
@@ -1,3 +1,284 @@
...
+- **Multiple Backup Locations for Improved Business Continuity:** As KKP Admin, you can now configure multiple backup destinations from the UI Admin Panel. Users can select daily, weekly, monthly or customized backup options.
+
+### Breaking Changes
+
+- ACTION REQUIRED: When upgrading from older KKP versions, additional flag --migrate-nginx-ingress is required to perform upgrade of nginx-ingress-controller. Ingress will be briefly unavailable during this process ([#8333](https://github.com/kubermatic/kubermatic/issues/8333))
+- ACTION REQUIRED: Set the default Service Accounts automountServiceAccountToken property to false for all KKP provided namespaces and kube-system ([#8344](https://github.com/kubermatic/kubermatic/issues/8344))
```
Comment (themue) [inline CHANGELOG.md]: `automountServiceAccountToken`

File: CHANGELOG.md [hunk 14]
```diff
@@ -1,3 +1,284 @@
...
+
+### New and Enhanced Features
+
+#### External Cluster
+- New endpoint to get external cluster upgrades `GET /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/upgrades` ([#8115](https://github.com/kubermatic/kubermatic/issues/8115))
+- New endpoint to list external cluster machine deployments `GET /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/machinedeployments` ([#8200](https://github.com/kubermatic/kubermatic/issues/8200))
+- Add PATCH endpoint for external cluster `PATCH /api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}`([#8135](https://github.com/kubermatic/kubermatic/issues/8135))
+- New endpoint to list External MD Metrics: `/api/v2/projects/{project_id}/kubernetes/clusters/{cluster_id}/machinedeployments/{machinedeployment_id}/nodes/metrics` ([#8239](https://github.com/kubermatic/kubermatic/issues/8239))
+- Manage AKS Clusters natively. Extend external cluster functionality to import existing AKS cluster ([#8244](https://github.com/kubermatic/kubermatic/issues/8244))
+- Create AKS nodePool ([#8399](https://github.com/kubermatic/kubermatic/issues/8399))
```
Comment (themue) [inline CHANGELOG.md]: NodePool

File: CHANGELOG.md [hunk 15]
```diff
@@ -1,3 +1,284 @@
...
+- Create missing ClusterRole 'cert-manager-edit' for cert-manager ([#8554](https://github.com/kubermatic/kubermatic/issues/8554))
+- Fixes a bug where `$$` in the environment-variables for machine-controller was interpreted in the Kubernetes Manifest and caused machine-controller to be unable to deploy resources, when for e.g. the password contains two consecutive `$` signs ([#7984](https://github.com/kubermatic/kubermatic/issues/7984))
+- kubeadm-config is updated based on Kubernetes version ([#8149](https://github.com/kubermatic/kubermatic/issues/8149))
+- Fix kubermatic-installer not properly updating CRDs ([#8552](https://github.com/kubermatic/kubermatic/issues/8552))
+- Fix PodDisruptionBudgets for master/seed-controller-manager blocking node rotations ([#8672](https://github.com/kubermatic/kubermatic/issues/8672))
+- Fix clusters being occasionally stuck in deletion because seed-level RBAC resources were deleted too early ([#8744](https://github.com/kubermatic/kubermatic/issues/8744))
+
+### Updates
+- update nginx-ingress-controller to 1.0.0 to support Kubernetes 1.22 master/seed clusters ([#7845](https://github.com/kubermatic/kubermatic/issues/7845))
```
Comment (themue) [inline CHANGELOG.md]: All "- update"  to upper-case.

File: CHANGELOG.md [hunk 16]
```diff
@@ -1,3 +1,284 @@
...
+- Bump operating-system-manager to v0.3.2 ([#8570](https://github.com/kubermatic/kubermatic/issues/8570))
+- Upgrade machine controller to 1.36.2 ([#8233](https://github.com/kubermatic/kubermatic/issues/8233))
+- machine-controller is updated to v1.41.0 ([#8590](https://github.com/kubermatic/kubermatic/issues/8590))
+- machine-controller updated to v1.41.1  
+  operating-system-manager updated to v0.3.6 ([#8693](https://github.com/kubermatic/kubermatic/issues/8693))
+- Update machine-controller to v1.42.0 ([#8731](https://github.com/kubermatic/kubermatic/issues/8731))
+- Update machine controller to 1.42.1 ([#8757](https://github.com/kubermatic/kubermatic/issues/8757))
+- Removed support for Kubernetes 1.19 ([#8167](https://github.com/kubermatic/kubermatic/issues/8167))
+- Add support for Kubernetes 1.22.5, 1.21.8, and 1.20.14  
+  Automatically upgrade clusters running Kubernetes 1.21 to 1.21.8 to include fixes for CVE-2021-44716 and CVE-2021-44717
```
Comment (themue) [inline CHANGELOG.md]: Aren't these also dashpoints?

File: CHANGELOG.md [hunk 17]
```diff
@@ -1,3 +1,284 @@
...
+- Add account audits support to the admin settings ([#3782](https://github.com/kubermatic/dashboard/issues/3782))
+- Allow configuring AWS AssumeRole credentials for user clusters ([#3811](https://github.com/kubermatic/dashboard/issues/3811))
+- Add Konnectivity support ([#3822](https://github.com/kubermatic/dashboard/issues/3822))
+- Add support for RHEL and Flatcar to the VSphere provider. Make RHEL subscription manager fields optional ([#3824](https://github.com/kubermatic/dashboard/issues/3824))
+- Disable dialog select form until existing template is selected ([#3836](https://github.com/kubermatic/dashboard/issues/3836))
+- Fix a bug where OPA data was not retrieved when there were no machine deployments in the cluster ([#3851](https://github.com/kubermatic/dashboard/issues/3851))
+- Add support for Openstack authentication with `project name` and  `project id`.  `tenant name`  and `tenant id`  are deprecated and have been renamed to `project name` and  `project id` in the UI ([#3854](https://github.com/kubermatic/dashboard/issues/3854))
+- Added audit policy preset picker to the wizard ([#3862](https://github.com/kubermatic/dashboard/issues/3862))
```
Comment (themue) [inline CHANGELOG.md]: All past tenses into today ("Added" to "Add").

File: CHANGELOG.md (hunk 10 above)
Comment (vgramer) [reply CHANGELOG.md:27]: i added right after `--migrate-nginx-ingress` for more visibility

**Review by themue — COMMENTED** [review]
Some cosmetic changes.

File: CHANGELOG.md (hunk 12 above)
Comment (vgramer) [reply CHANGELOG.md]: done :)

File: CHANGELOG.md (hunk 11 above)
Comment (vgramer) [reply CHANGELOG.md]: done :)

File: CHANGELOG.md [hunk 18]
```diff
@@ -1,3 +1,285 @@
...
+
+#### VMware vSphere
+- Support latest vSphere Cloud Controller Manager and  CSI driver for Kubernetes 1.22 ([#8505](https://github.com/kubermatic/kubermatic/issues/8505))
+
+#### Google Cloud Platform (GCP)
+- Manage GKE Clusters natively. Extend external cluster functionality to import existing GKE cluster ([#8046](https://github.com/kubermatic/kubermatic/issues/8046))
+
+#### KubeVirt
+- Add API endpoints for Kubevirt VirtualMachineInstancePresets and StorageClasses ([#8441](https://github.com/kubermatic/kubermatic/issues/8441))
+- Support for KubeVirt CSI driver ([#8416](https://github.com/kubermatic/kubermatic/issues/8416))
```
Comment (hdurand0710) [inline CHANGELOG.md:154]: Should we mention a link to the already known bug #8772 ?

File: CHANGELOG.md (hunk 16 above)
Comment (vgramer) [reply CHANGELOG.md]: done :)

File: CHANGELOG.md (hunk 17 above)
Comment (vgramer) [reply CHANGELOG.md]: done :)

File: CHANGELOG.md (hunk 14 above)
Comment (vgramer) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 15 above)
Comment (vgramer) [reply CHANGELOG.md]: done

File: CHANGELOG.md [hunk 19]
```diff
@@ -1,3 +1,285 @@
+# Kubermatic 2.19
+## [v2.19.0](https://github.com/kubermatic/kubermatic/releases/tag/v2.19.0)
+
+Before upgrading, make sure to read the [general upgrade guidelines](https://docs.kubermatic.com/kubermatic/v2.19/upgrading/guidelines/). Consider tweaking `seedControllerManager.maximumParallelReconciles` to ensure usercluster reconciliations will not cause resource exhausting on seed clusters.
```
Comment (sankalp-r) [inline CHANGELOG.md]: In place of `exhausting `,  `exhaustion` can be used.

File: CHANGELOG.md (hunk 13 above)
Comment (vgramer) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 8 above)
Comment (vgramer) [reply CHANGELOG.md]: done

File: CHANGELOG.md [hunk 20]
```diff
@@ -1,3 +1,285 @@
...
+- nginx-ingress-controller chart has been updated to use upstream chart from https://kubernetes.github.io/ingress-nginx as a dependencyACTIONS REQUIRED:* entire nginx-ingresss-controller configuration is moved to a subkey in values file: `nginx.controller`* option to run as a daemonset removed (`nginx.asDaemonSet`)* option to schedule on master nodes removed (`nginx.ignoreMasterTaint`) and a way to reconfigure it has been added to the `values.yaml` file ([#8277](https://github.com/kubermatic/kubermatic/issues/8277))
+- Enhance apiserver-to-node throughput by relaxing OpenVPN & envoy-agent CPU limits ([#8102](https://github.com/kubermatic/kubermatic/issues/8102))
+- Change KonnectivityEnabled flag to a pointer in ClusterNetworkingConfig to make defaulting easier ([#8103](https://github.com/kubermatic/kubermatic/issues/8103))
+
+#### Cilium
+- Add Cilium CNI addon ([#7853](https://github.com/kubermatic/kubermatic/issues/7853))
+- Add ebpf proxy mode support for Cilium CNI ([#7861](https://github.com/kubermatic/kubermatic/issues/7861))
+- Add Cilium Hubble addon for CNI network observability ([#8548](https://github.com/kubermatic/kubermatic/issues/8548))
+
```
Comment (moadqassem) [inline CHANGELOG.md:126]: Can we add that, Cillium on RHEL with Azure using iptables as proxy mode is not supported

File: CHANGELOG.md [hunk 21]
```diff
@@ -1,3 +1,285 @@
...
+### Cloud Providers
+
+#### Google Cloud Platform (GCP)
+- Fix missing validation for GCE datacenter zone_suffixes ([#8525](https://github.com/kubermatic/kubermatic/issues/8525))
+
+#### VMware vSphere
+- Fix vSphere clusters getting stuck after CSI migration due to bad ValidatingWebhookConfiguration reconciling ([#8738](https://github.com/kubermatic/kubermatic/issues/8738))
+
+### Mics
```
Comment (themue) [inline CHANGELOG.md]: s/Mics/Misc/g

File: CHANGELOG.md [hunk 22]
```diff
@@ -1,3 +1,285 @@
...
+- **Enhanced Control Plane Networking with Konnectivity Support:** To harness the power of Cilium with eBPF, KKP 2.19 comes with added Konnectivity support. It provides TCP level proxy for the control plane (seed cluster) to worker nodes (user cluster) communication. It is based on the upstream [apiserver-network-proxy project](https://github.com/kubernetes-sigs/apiserver-network-proxy) and replaces the older KKP-specific solution based on OpenVPN and network address translation.
+- **Multiple Backup Locations for Improved Business Continuity:** As KKP Admin, you can now configure multiple backup destinations from the UI Admin Panel. Users can select daily, weekly, monthly or customized backup options.
+
+### Breaking Changes
+
+- ACTION REQUIRED: When upgrading from older KKP versions, additional flags:
+  - `--migrate-nginx-ingress` is required to perform upgrade of nginx-ingress-controller. Ingress will be briefly unavailable during this process ([#8333](https://github.com/kubermatic/kubermatic/issues/8333))
```
Comment (wurbanski) [inline CHANGELOG.md]: The flag has been renamed to `--migrate-upstream-nginx-ingress`

File: CHANGELOG.md [hunk 23]
```diff
@@ -1,3 +1,285 @@
...
+- kubeadm-config is updated based on Kubernetes version ([#8149](https://github.com/kubermatic/kubermatic/issues/8149))
+- Fix kubermatic-installer not properly updating CRDs ([#8552](https://github.com/kubermatic/kubermatic/issues/8552))
+- Fix PodDisruptionBudgets for master/seed-controller-manager blocking node rotations ([#8672](https://github.com/kubermatic/kubermatic/issues/8672))
+- Fix clusters being occasionally stuck in deletion because seed-level RBAC resources were deleted too early ([#8744](https://github.com/kubermatic/kubermatic/issues/8744))
+
+### Updates
+- Update nginx-ingress-controller to 1.0.0 to support Kubernetes 1.22 master/seed clusters ([#7845](https://github.com/kubermatic/kubermatic/issues/7845))
+- Update nginx-ingress-controller to 1.0.2 ([#7875](https://github.com/kubermatic/kubermatic/issues/7875))
+- Update Kubernetes Dashboard to v2.4.0 ([#8172](https://github.com/kubermatic/kubermatic/issues/8172))
+- update Dex to 2.30.0 ([#7846](https://github.com/kubermatic/kubermatic/issues/7846))
```
Comment (themue) [inline CHANGELOG.md]: %s/- update/- Update/g

File: CHANGELOG.md (hunk 19 above)
Comment (vgramer) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 22 above)
Comment (vgramer) [reply CHANGELOG.md]: done

**Review by themue — COMMENTED** [review]
Two more minor changes.

File: CHANGELOG.md [hunk 24]
```diff
@@ -1,3 +1,285 @@
...
+
+### Cloud Providers
+
+#### Google Cloud Platform (GCP)
+- Fix missing validation for GCE datacenter zone_suffixes ([#8525](https://github.com/kubermatic/kubermatic/issues/8525))
+
+#### VMware vSphere
+- Fix vSphere clusters getting stuck after CSI migration due to bad ValidatingWebhookConfiguration reconciling ([#8738](https://github.com/kubermatic/kubermatic/issues/8738))
+
+### Mics
```
Comment (sankalp-r) [inline CHANGELOG.md:221]: One more typo for `Misc`.

File: CHANGELOG.md (hunk 21 above)
Comment (vgramer) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 23 above)
Comment (vgramer) [reply CHANGELOG.md]: done

File: CHANGELOG.md [hunk 25]
```diff
@@ -1,3 +1,285 @@
...
+- **Multiple Backup Locations for Improved Business Continuity:** As KKP Admin, you can now configure multiple backup destinations from the UI Admin Panel. Users can select daily, weekly, monthly or customized backup options.
+
+### Breaking Changes
+
+- ACTION REQUIRED: When upgrading from older KKP versions (before v2.19.0), additional flags:
+  - `--migrate-upstream-nginx-ingress` is required to perform upgrade of nginx-ingress-controller. Ingress will be briefly unavailable during this process ([#8333](https://github.com/kubermatic/kubermatic/issues/8333))
+  - `--migrate-upstream-cert-manager` is required to perform to perform the migration of the `cert-manager`. During the upgrade, the chart is uninstalled completely so there is a short time when certificates will not be renewed.([#8392](https://github.com/kubermatic/kubermatic/issues/8392))
```
Comment (sankalp-r) [inline CHANGELOG.md]: `to perform` repeated twice.

**Pull request merged**